src/debugger/adapter/DebugAdapter.js

```diff
@@ -247,6 +254,12 @@ class PrepackDebugSession extends LoggingDebugSession {
       }
     );
   }
+
+  stepInRequest(response: DebugProtocol.StepInResponse, args: DebugProtocol.StepInArguments): void {
```

Comment:
Add comment "// override" for all these methods so that reviewer can easily see how this method is used/called.